src/js/components/TextArea/index.d.ts

```diff
@@ -1,6 +1,8 @@
 import * as React from "react";
+import {A11yTitleType} from "../../utils";
```

Comment:
```suggestion
import { A11yTitleType } from "../../utils";
```
I've filed this issue https://github.com/grommet/grommet/issues/4263